spec/datetimeformat.html

```diff
@@ -732,6 +761,7 @@ <h1>Properties of Intl.DateTimeFormat Instances</h1>
       <li>[[NumberingSystem]] is a String value with the `"type"` given in Unicode Technical Standard 35 for the numbering system used for formatting.</li>
       <li>[[TimeZone]] is a String value with the IANA time zone name of the time zone used for formatting.</li>
       <li>[[Weekday]], [[Era]], [[Year]], [[Month]], [[Day]], [[Hour]], [[Minute]], [[Second]], [[TimeZoneName]] are each either *undefined*, indicating that the component is not used for formatting, or one of the String values given in <emu-xref href="#table-datetimeformat-components"></emu-xref>, indicating how the component should be presented in the formatted output.</li>
+      <li>[[FractionalSecondDigits]] is either *undefined* or a non-negative integer Number value indicating the fraction digits to be used for fractional second. Numbers will be rounded or padded with trailing zeroes if necessary. </li>
```

Comment:
@anba Do you think I should change this to "either *undefined* or a positive integer Number value "?

Reply:
Yes. Maybe even "[...] a positive, non-zero integer Number value" to make clear that 0 isn't allowed. ("strictly positive" is more concise than "positive and non-zero", but the latter doesn't require to know the term "strictly positive".)